### DISCUSSION TITLE
Week 3 - Assignment Thoughts

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#70] [Week 3 - Discussions]
In all reality, I really liked the FreeCodeCamp JavaScript approach. I felt like it was much easier to understand than the exercism approach. In addition, I think it would have been better to use this assignment prior to doing exercism as it helped me understand more about how to do the assignments through exercism. 

This week's HTML/CSS approach was fairly simple and straightforward. I liked working on the website and found it effective at teaching CSS inline and in a separate document.  I also thought it worked well with the froggy game. I honestly didn't understand the purpose of the CSS Diner. It was partially confusing, but it might be a little better now that I went through some practice HTML apps and learned more of the basics.

Another thing that really helped me understand HTML and CSS was creating my own HTML and CSS files and practicing creating a site on my own and seeing how it changes when I open it in a browser.

**Comment by kassyoveson** [comment]
I agree that we should have done FreeCodeCamp before exercism. I learned more when I could see the changes. No matter what program I use, as a beginner, they are all confusing!

**Reply by metasean** [reply]
@kassyoveson When you hit something confusing, post a question in the discussion.  You are likely not the only student who is confused by certain task (e.g. Derek and the purpose of CSS Diner).  fwiw: I really want y'all asking and answering questions yourselves, because answering the questions can be as beneficial as getting the answer.

**Comment by metasean** [comment]
@derekscholes - You might want to revisit CSS Diner in a few weeks.
